web-components/src/components/tabs/scss/tabs.scss

```diff
@@ -92,6 +94,29 @@
   padding: $tab-base-indent;
 }
 
+md-tab {
+  md-icon {
+    margin-right: 8px;
+  }
+}
+
+md-menu-overlay {
+  md-tab {
+    md-tooltip {
+      width: 210px;
+      position: absolute;
+
+      .text-ellipsis {
+        width: 210px;
```

Comment:
why hard-coded width inside a component? is there a way if any client code wants to change the width?